lib/client.js

```diff
@@ -51,6 +51,12 @@ Client.prototype.join = function (browserWindow, options, cb) {
     }
     browserWindow = getBrowserWindow();
   }
+  // handle old boolean verbose values
+  if (this.opt.verbose === true) {
+    this.opt.verbose = 2;
+  } else if (this.opt.verbose === false) {
+    this.opt.verbose = 1;
+  }
```

Comment:
Change `this.opt.verbose` to `this.opt.logLevel`. And, use constant names instead of numbers.